resources/views/InventoryList/InventoryList.tsx

```diff
@@ -123,6 +151,16 @@ const InventoryList = ({
             key: 'stock',
             dataIndex: 'stock',
         },
+        {
+            title: 'Action',
+            key: 'action',
+            dataIndex: 'action',
+            render: (text: any, record: any) => (
+              <Space size='middle'>
+                <Button danger type='text' icon={<DeleteOutlined />} size='large' onClick={() => {deleteItemModal(record)}} />
```

Comment:
we can add color theme to this icon

Reply:
Right now the icon is red, would orange be more appropriate?

Reply:
sure, get the color code from figma